src/main/java/com/example/contactsservice/Service/ContactService.java

```diff
@@ -0,0 +1,86 @@
+package com.example.contactsservice.Service;
+
+import com.example.contactsservice.DTO.ContactDTO;
+import com.example.contactsservice.Entity.ContactEntity;
+import com.example.contactsservice.Exception.NotFound;
+import com.example.contactsservice.Repository.ContactRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class ContactService {
```

Comment:
Instead of using service class directly, please create an interface with name 'ContactService' and then create a class with name 'ContactServiceImpl' and implement the interface 'ContactService'